lib/main_common.pm

```diff
@@ -192,6 +192,11 @@ sub setup_env {
         my $format_dasd = get_var('S390_DISK') || get_var('UPGRADE') || get_var('ENCRYPT_ACTIVATE_EXISTING') ? 'never' : 'pre_install';
         set_var('FORMAT_DASD', get_var('FORMAT_DASD', $format_dasd));
     }
+    # This is for 12-sp5 project specific flavor Migration-from-SLE12-SP5-to-SLE15-SPx, this flavor belong 12sp5 test group but the real
+    # action is migration from 12-sp5 to 15-sp1, so we need change VERSION to 15-SP1 before the test case start
+    if (check_var('FLAVOR', 'Migration-from-SLE12-SP5-to-SLE15-SPx')) {
+        set_var('VERSION', '15-SP1');
+    }
```

Comment:
I am really no friend of service pack specific code. If you really must have a version-specific flavor (which also does not look correct to me) then you can read out the version from there. https://openqa.suse.de/admin/products shows that "Migration-from-SLE12-SP5-to-SLE15-SPx" already *has* VERSION=15-SP1 so why do you need to set the version? Maybe you actually need to set 12-SP5 here? If that is the case you can parse the version from the flavor string dynamically.

Reply:
@okurz  Migration-from-SLE12-SP5-to-SLE15-SPx should has VERSION=12-SP5(Currently you see version=12-sp5 is only for debugging purpose), so i need this PR to change version to 15-SP1 for real test cases.